.cirrus.yml

```diff
@@ -66,6 +66,20 @@ task:
   test_script:
     - set -e
     - curl -Ls https://raw.githubusercontent.com/ROM-builders/temporary/main/test.sh | bash
+
+  check_script:
+    - set -e
+    - rom_name=$(grep init $CIRRUS_WORKING_DIR/build_rom.sh -m 1 | cut -d / -f 4)
+    - branch_name=$(grep init $CIRRUS_WORKING_DIR/build_rom.sh | awk -F "-b " '{print $2}' | awk '{print $1}')
+    - rom_name=$rom_name-$branch_name
+    - z=$(grep 'clang' ~/roms/$rom_name/.repo/local_manifests/local_manifest.xml -m1 || true)
```

Comment:
it should be something like grep clang word in local manifest of person X then get the whole line stating the path of the clang and then remove that clang from the path which it got by greping

Reply:
dont restricte clang because there is some kernel need ld then they add custom clang with ld support ( is not supported on lineage havoc and aex) to build rom
the error:
clang-9: error: unable to execute command: Executable "ld" doesn't exist!
clang-9: error: linker command failed with exit code 1 (use -v to see invocation)
/home/cirrus/roms/LineageOS-lineage-17.1/kernel/samsung/exynos7870/scripts/fips_crypto_hmac.sh : /home/cirrus/roms/LineageOS-lineage-17.1/prebuilts/clang/host/linux-x86/clang-r353983c1/bin/clang returned error

Reply:
> dont restricte clang because there is some kernel need ld then they add custom clang with ld support ( is not supported on lineage havoc and aex) to build rom the error: clang-9: error: unable to execute command: Executable "ld" doesn't exist! clang-9: error: linker command failed with exit code 1 (use -v to see invocation) /home/cirrus/roms/LineageOS-lineage-17.1/kernel/samsung/exynos7870/scripts/fips_crypto_hmac.sh : /home/cirrus/roms/LineageOS-lineage-17.1/prebuilts/clang/host/linux-x86/clang-r353983c1/bin/clang returned error

the script is like you will sync the clang but when someone else is building same ROM it gets back to it's default state and when you will build again it will sync your clang again from your local manifest, i hope you get it

Reply:
ah ok